### DISCUSSION TITLE
generating-editorconfig-files-automatically-using-intellicode/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#387] [Announcements]
# generating-editorconfig-files-automatically-using-intellicode/

In this post I show how to install the IntelliCode extension in Visual Studio, and how to use it to generate an EditorConfig file from your code.

https://andrewlock.net/generating-editorconfig-files-automatically-using-intellicode/

**Comment by NumbGnat** [comment]
I'm sorry to resurrect this post, but I came here looking for the same thing you found...

I find it frustrating trying to reconfigure Visual Studio to match random GitHub projects across varying developers. I was looking for an easy way to generate an `.editorconfig` file, based on the original author's preferences.  So when I make enhancements and generate a pull request, my code matches the rest of the project.

I am running Visual Studio 2022 (Community), which doesn't support this extension. And that's fine, because most of what IntelliCode does/did, is now baked into Visual Studio.  The problem I have is that the [Add]->[New Item...]->[EditorConfig (IntelliCode)] functionality doesn't seem to have been absorbed into Visual Studio.

Any ideas where I might find a tool to do that these days?
